feconf.py

```diff
@@ -63,7 +56,7 @@
 
 # Choose production templates folder if minification flag is used or
```

Comment:
Choose production templates folder when we are in production mode.

Reply:
Done